src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -264,80 +273,19 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
-                                  ${!supportsFeature(
-                                    stateObj,
-                                    SUPPORT_PREVIOUS_TRACK
-                                  )
-                                    ? ""
-                                    : html`
-                                        <paper-icon-button
-                                          icon="hass:skip-previous"
-                                          .action=${"media_previous_track"}
-                                          @click=${this._handleClick}
-                                        ></paper-icon-button>
-                                      `}
-                                  ${(stateObj.state !== "playing" &&
-                                    !supportsFeature(
-                                      stateObj,
-                                      SUPPORTS_PLAY
-                                    )) ||
-                                  stateObj.state === UNAVAILABLE ||
-                                  (stateObj.state === "playing" &&
-                                    !supportsFeature(stateObj, SUPPORT_PAUSE) &&
-                                    !supportsFeature(stateObj, SUPPORT_STOP))
-                                    ? ""
-                                    : html`
-                                        <paper-icon-button
-                                          class="playPauseButton"
-                                          .icon=${stateObj.state !== "playing"
-                                            ? "hass:play"
-                                            : supportsFeature(
-                                                stateObj,
-                                                SUPPORT_PAUSE
-                                              )
-                                            ? "hass:pause"
-                                            : "hass:stop"}
-                                          .action=${"media_play_pause"}
-                                          @click=${this._handleClick}
-                                        ></paper-icon-button>
-                                      `}
-                                  ${!supportsFeature(
-                                    stateObj,
-                                    SUPPORT_NEXT_TRACK
-                                  )
-                                    ? ""
-                                    : html`
-                                        <paper-icon-button
-                                          icon="hass:skip-next"
-                                          .action=${"media_next_track"}
-                                          @click=${this._handleClick}
-                                        ></paper-icon-button>
-                                      `}
-                                </div>
-                              `}
+                        <div class="playback-controls">
```

Comment:
Update the CSS class to be `.playback-controls` or change this back to `.controls`